src/Arabic.php

```diff
@@ -3872,11 +3872,15 @@ public function volc($olc, $codeLength = 10)
      * @param string  $plural3  Plural form 3 (e.g., عناصر). If NULL [default] retrive from internal JSON dataset.
      * @param string  $plural4  Plural form 4 (e.g., عنصرا). If NULL [default] retrive from internal JSON dataset.
      *
-     * @return string Proper plural form of the given singular form
+     * @return string|\Exception Proper plural form of the given singular form, throw exiption if the $singular is not pre defiend and $plural is not passed
      * @author Khaled Al-Sham'aa <khaled@ar-php.org>
      */
     public function arPlural($singular, $count, $plural2 = null, $plural3 = null, $plural4 = null)
     {
+        if(!array_key_exists($singular, $this->arPluralsForms)){
```

Comment:
This condition with throw an exception even if the developer passing the plural forms in explicit way (i.e. plural2, plural3, and plural4). Instead, I believe it should be something like this:
`if(!array_key_exists($singular, $this->arPluralsForms) && (is_null($plural2) || is_null($plural3) || is_null($plural4))){`